scripts/projects.js

```diff
@@ -1,99 +1,46 @@
-'use strict';
+"use strict";
 
 const basicCards = document.getElementById('basicCards');
 const moreCards = document.getElementById('moreCards');
 
 const projectDetails = [
 	{
-		coverImage: 'https://source.unsplash.com/300x225/?wave',
-		projectName: 'DeepPixel',
+		coverImage: 'https://static.toiimg.com/photo/msid-70093419,width-300,height-225.cms',
```

Comment:
I would suggest to avoid using external sources for loading assets and instead have them locally